lib/user.rb

```diff
@@ -0,0 +1,34 @@
+require "bcrypt"
+
+class User
+  attr_reader :id, :email
+  def initialize(id:, email:)
+    @id = id
+    @email = email
+  end
+
+  def self.create(email:, password:)
+    encrypted_password = BCrypt::Password.create(password)
```

Comment:
Really great work on encrypting the passwords before being stored onto the database!